brave/brave-core
# Add New tab page options to settings

With this, user can select which page should be opened in NTP.

<img width="770" alt="Screen Shot 2020-10-23 at 5 41 29 PM" src="https://user-images.githubusercontent.com/6786187/96976677-13980500-1557-11eb-87df-7ddc4208da83.png">
<img width="843" alt="Screen Shot 2020-10-23 at 5 41 40 PM" src="https://user-images.githubusercontent.com/6786187/96976681-1561c880-1557-11eb-81d9-e12048cee593.png">

dark mode blank page:
![Screenshot from 2020-10-28 10-47-00](https://user-images.githubusercontent.com/6786187/97380702-f7a3b300-190a-11eb-878a-c9e5dad3bcd4.png)



Resolves https://github.com/brave/brave-browser/issues/2999

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
`npm run test brave_unit_tests -- --filter=*BraveNewTabTest*`

1. Choose Dashboard...
2. Create NTP by new tab button, shortcut(ex, CMD+T or new tab from app menu)
3. Check brave NTP is opened
4. Choose HomePage
5. If newtab is homepage is on, create NTP is brave NTP
6. If newtab is homepage is off, configured page is opened
7. Choose blankpage
8. Check about:blank is loaded for NTP

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
